src/components/MessageLivestream.js

```diff
@@ -87,12 +87,16 @@ class MessageLivestream extends Component {
 				}
 
 				<div key={message.id} className={`ch-msg-padding ${msgContainerPos}`}>
-					<div className={`ch-message-container`}>
-						<div title={message.owner.displayName} className="ch-message-owner-avatar" style={{backgroundImage:`url(${ownerProfileImageUrl})`}}></div>
+					<div className={`ch-msg-container ch-msg-container-livestream`}>
+
+						<UserIcon user={message.owner} className="ch-message-owner-avatar"></UserIcon>
+
 						<div className={`ch-msg-content ch-msg-content__livestream`}>
 								{ message.body && 
 									<div className={`ch-text-message`}>
-										<div className="ch-message-owner-name">{message.owner.displayName}</div>
+										<div className="ch-message-owner-name">{message.owner.displayName}
```

Comment:
@abhishekbadoni-bigstep are not we using `avatar` as we were supposed to use avatar here also.

Reply:
We are using avatar here also.